src/OpenTelemetry/Trace/DefaultTracing.cs

```diff
@@ -16,49 +16,33 @@
 
 namespace OpenTelemetry.Trace
 {
-    using System.Runtime.CompilerServices;
     using OpenTelemetry.Internal;
     using OpenTelemetry.Trace.Config;
     using OpenTelemetry.Trace.Export;
     using OpenTelemetry.Trace.Internal;
 
     /// <summary>
-    /// Class that manages a global instance of the <see cref="Tracer"/>.
+    /// Class that defines the default global instances of the <see cref="Tracing"/> APIs.
     /// </summary>
-    public sealed class Tracing
+    public static class DefaultTracing
     {
-        private static Tracing tracingValue = new Tracing();
-        private static Tracer tracer;
-
-        internal Tracing()
+        /// <summary>
+        /// Initializes the default tracing implementation.
+        /// </summary>
+        public static void Init()
```

Comment:
I don't think it is acceptable to expect libraries to check if tracer has changed after they first requested it. 
So We should come up with the approach where they will call some proxy that will represent either noop or real tracer.

Reply:
I've pushed a proxy instance that won't change. `Init` is able to swap the actual `ITracer` and `ISpanExporter`. I'll resolve this one, please advise on that change.